compiler/noirc_evaluator/src/ssa/ssa_gen/mod.rs

```diff
@@ -620,6 +621,47 @@ impl<'a> FunctionContext<'a> {
         Ok(Self::unit_value())
     }
 
+    /// Codegens a while loop, creating three new blocks in the process.
+    /// The return value of a while is always a unit literal.
+    ///
+    /// For example, the loop `while cond { body }` is codegen'd as:
+    ///
+    /// ```text
+    ///   jmp while_entry()
+    /// while_entry:
+    ///   v0 = ... codegen cond ...
+    ///   jmpif v0, then: while_body, else: while_end  
+    /// while_body():
+    ///   v3 = ... codegen body ...
+    ///   br while_entry()
+    /// while_end():
+    ///   ... This is the current insert point after codegen_for finishes ...
```

Comment:
```suggestion
    ///   jmp while_entry()
    /// while_end():
    ///   ... This is the current insert point after codegen_while finishes ...
```

Reply:
Thanks! That `br` is from having worked with LLVM 😅